app/views/pages/accessibility.erb

```diff
@@ -0,0 +1,85 @@
+<div class="govuk-grid-row govuk-body">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Accessibility statement</h1>
+    <p class="govuk-body">This accessibility statement applies to Complete
+      conversions, transfers and changes.</p>
+    <p class="govuk-body">This website is run by the Department for Education
+      (DfE). We want as many people as possible to be able to use this website.
+      For example, that means you should be able to:</p>
+    <ul>
+      <li>change colours, contrast levels and fonts</li>
+      <li>zoom in up to 300% without the text spilling off the screen</li>
+      <li>navigate most of the website using just a keyboard</li>
+      <li>navigate most of the website using speech recognition software</li>
+      <li>listen to most of the website using a screen reader (including the
+        most recent versions of JAWS, NVDA and VoiceOver)</li>
+    </ul>
+    <p class="govuk-body">We've also made the website text as simple as possible
+      to understand.</p>
+    <p class="govuk-body"><a href="https://mcmw.abilitynet.org.uk/" target="_blank" rel="noopner noreferrer">
+      AbilityNet (opens in a new tab)</a> has advice on making your device
+      easier to use if you have a disability.
+    <h2 class="govuk-heading-l">How accessible this website is</h2>
+    <p class="govuk-body">This website is partially compliant with the
```

Comment:
At risk of veering into content questions, is this the best phrasing? It's incredibly difficult to make a website which _isn't_ partially compliant, so this sounds like weasel words to me.

Reply:
Yeah its not super great but it is what DfE suggest/want. When we pass the audit we will change to one that says we are compliant, so this is temporary.